frontend/cypress/integration/multiProject.spec.js

```diff
@@ -0,0 +1,291 @@
+import faker from 'faker'
+const syncedRepoUrlMultiProjects =
+  'https://github.com/kitspace-forks/DIY_particle_detector'
+const syncedRepoUrl = 'https://github.com/kitspace-forks/CH330_Hardware'
+const multiProjectsNames = ['alpha-spectrometer', 'electron-detector']
+const multiProjectsRepoName = 'DIY_particle_detector'
+const normalRepoName = 'CH330_Hardware'
+
+describe('Render project cards', () => {
+  before(() => {
+    /*
+     * The purpose of this isn't actually visiting the homepage.
+     * Sometimes, the frontend has a slow startup time which results in a random failure.
+     */
+    cy.visit('/')
+  })
+
+  it('should render a card for each multiproject', () => {
+    const username = faker.name.firstName()
+    const email = faker.internet.email()
+    const password = '123456'
+
+    const repoName = syncedRepoUrlMultiProjects.split('/').slice(-1).toString()
+
+    cy.intercept('http://gitea.kitspace.test:3000/user/kitspace/**').as('sign_in')
```

Comment:
I saw you remove the others but there's still more `cy.intercepts` that I don't understand. :grimacing:

Reply:
Because it's used in line 30 to wait for the `sign_in` request to finish. It should wait for a UI change but this segment is used in almost all the tests. I will fix it for all tests but not in this PR.